lib/fastlane/fast_file.rb

```diff
@@ -28,9 +28,9 @@ def parse(data)
           eval(data) # this is okay in this case
           # rubocop:enable Lint/Eval
         rescue SyntaxError => ex
-          if ex.to_s.include? "‘"
+          if data.to_s.force_encoding("utf-8").include? "\u2018"
```

Comment:
In my case the ex didn't include any of that char when running the test. Instead it was something like ????. That was the only way to make it to work in my case.